composer.json

```diff
@@ -70,7 +71,8 @@
         "laravel": {
             "providers": [
                 "MongoDB\\Laravel\\MongoDBServiceProvider",
-                "MongoDB\\Laravel\\MongoDBBusServiceProvider"
+                "MongoDB\\Laravel\\MongoDBBusServiceProvider",
+                "MongoDB\\Laravel\\Scout\\ScoutServiceProvider"
```

Comment:
Would "ScoutServiceProvider" likely conflict with other short class names? There looks to be redundancy in both suggestions here, but I'll defer to whatever you think is most consistent.

It's been a couple of weeks since I picked up the Telescope PR but IIRC I opted to use a namespace there as there would be multiple classes. Seemed preferable than keeping a service provider in a separate namespace from other classes associated with the feature.